snippets/r.json

```diff
@@ -118,15 +118,17 @@
     "shinymod": {
         "prefix": "shinymod",
         "body": [
-            "${1:name}_UI <- function(id) {",
+            "${1:name}UI <- function(id) {",
             "  ns <- NS(id)",
             "  tagList(",
-            "    ${0}",
+            "    ${2}",
             "  )",
             "}",
             "",
-            "${1:name} <- function(input, output, session) {",
-            "  ",
+            "${1:name}Server <- function(id) {",
+            "  moduleServer(id, function(input, output, session) {",
+            "    ${3}"
```

Comment:
I believe the line containing the `${3}` needs a comma, else this will throw an error.